packages/block-editor/src/components/inserter/category-tabs/index.js

```diff
@@ -62,9 +75,29 @@ function CategoryTabs( {
 					key={ category.name }
 					tabId={ category.name }
 					focusable={ false }
-					className="block-editor-inserter__category-panel"
 				>
-					{ children }
+					<motion.div
```

Comment:
Is it better to have the motion div inside each tab, or could we put this outside `categories.map`?

Reply:
The tab won't mount its content until it is selected, so this is essentially one div until it mounts. It's possible it's better if we have it outside of it, but I'm concerned with that method since until we have a tabpanel selected, the semantics are off. I think this way with the motion div inside is safer semantically.

Reply:
The motion div only gets mounted when the category is mounted, so we don't generate loads of motion divs with this approach anyway.